c2corg_ui/static/js/apiservice.js

```diff
@@ -380,6 +387,37 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 
 
 /**
+ * @param {undefined | string} token
+ * @param {string} lang
+ * @param {boolean} isProfile
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.readFeed = function(token, lang, isProfile) {
+  var url;
+  var params = {'pl': lang};
+  if (token) params['token'] = token;
+
+  if (isProfile || this.auth_.isAuthenticated()) {
+    params['u'] = this.auth_.userData.id;
```

Comment:
``u`` is needed only for the user profile feed. Not for the auth homepage feed:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/views/feed.py#L73